ddl/ddl_api.go

```diff
@@ -5303,6 +5321,40 @@ func checkPlacementSpecs(specs []*ast.PlacementSpec) ([]*placement.Rule, error)
 			}
 		}
 
+		if err == nil {
+			switch spec.Tp {
+			case ast.PlacementAdd:
+				rule.Action = placement.RuleOpAdd
+			case ast.PlacementAlter:
+				rule.Action = placement.RuleOpAdd
```

Comment:
```suggestion
				rule.Action = placement.RuleOpAlter
```

Reply:
There are only `add` and `del`. It is not a typo